perun-core/src/main/java/cz/metacentrum/perun/core/api/VosManager.java

```diff
@@ -549,4 +549,25 @@
 	 * @throws VoNotExistsException if there is no vo with given id
 	 */
 	List<BanOnVo> getBansForVo(PerunSession sess, int voId) throws PrivilegeException, VoNotExistsException;
+
+	/**
+	 * For the given vo, creates sponsored members for each sponsored user who is a member
+	 * of the given vo.
```

Comment:
You should add info that all original sponors will be processed somehow too.

Reply:
Same for other javadocs of this method in different layers.

Reply:
Javadocs updated.